02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,49 @@
+const { people } = require("../data");
+
+const getPeople = (req, res) => {
+	res.json(people);
+};
+
+const addPerson = (req, res) => {
+	const lastIdIn_People = people[people.length - 1].id;
+	const { name } = req.body;
+	people.push({ id: lastIdIn_People + 1, name: name });
+	res.status(201).json({ success: true, name: name });
+};
+
+const getPerson = (req, res) => {
+	const persona = people.filter(
```

Comment:
This is a nitpick: `filter` could conceivably return multiple people from this search (because we're not using a real database to enforce the requirement that every person has a unique ID). I think `find` would be a better choice here, but it's not a big deal. 😊

Reply:
Thanks for the comment. I do want to think about that. Updating